src/app/about-us/mission-vision/mission-vision.component.html

```diff
@@ -0,0 +1,18 @@
+<div>
+  <div>
```

Comment:
These 2 divs don't serve a purpose, if you just want a container, delete at least one of them